### PR TITLE
build: update core to 1.0.0

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

File: docker/development/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN apt-get update -y \
 
 ## Open Space Toolkit ▸ Core
 
-ARG OSTK_CORE_VERSION="0.7.3"
+ARG OSTK_CORE_VERSION="1.0.0"
```
Comment (apaletta3) [inline docker/development/Dockerfile:33]: can't you just do this here?
```suggestion
ARG OSTK_CORE_VERSION="1.0"
```

**Review by apaletta3 — APPROVED** [review]
Lmk about my one comment, but good to merge

File: docker/development/Dockerfile (hunk 1 above)
Comment (vishwa2710) [reply docker/development/Dockerfile:33]: Nope. This doesn't have @kyle-cochran  fancy auto fetching script. Will add that in a separate MR.

Comment (apaletta3) [reply docker/development/Dockerfile:33]: ah right I forgot this is cpp and not python.

**Review by apaletta3 — APPROVED** [review]

**Pull request merged**